src/views/EditUserView.vue

```diff
@@ -224,7 +215,7 @@
         console.log(getData);
         getData.forEach(async (docData)=>{
           try{
-            await updateDoc(doc(firebaseStore, 'users_dev', docData.id),{
+            await updateDoc(doc(firebaseStore, 'user', docData.id),{
```

Comment:
do not put the collection name manually in the update doc. The collection names are stored in src/firebase/constants.js. Just import collections from there and use collections.user to get the right collection name.

Reply:
![image](https://user-images.githubusercontent.com/46050460/233805937-6f386bf3-522a-47a0-a8da-0fa9a0f4b93e.png)
![image](https://user-images.githubusercontent.com/46050460/233805948-304c2ec7-0ef5-4087-920d-68785b6cfd1d.png)
imported and using the constants.js data

Reply:
Yep. correct use of the collections constant.